flink-table/flink-table-common/src/main/java/org/apache/flink/table/types/extraction/DataTypeExtractor.java

```diff
@@ -335,7 +335,7 @@ private DataType extractDataTypeOrError(
         // early and helpful exception for common mistakes
         checkForCommonErrors(type);
 
-        // PREDEFINED
+        // PREDEFINED and DESCRIPTOR
         resultDataType = extractPredefinedType(template, type);
```

Comment:
given the fact that extract method here and for others contains type, should it be also renamed?